test/blocks/index.js

```diff
@@ -0,0 +1,73 @@
+import mocha from 'mocha';
```

Comment:
Should we use the `/** External/Internal dependecies **` comments (similar to Calypso)?